turbo/app/snapshots_cmd.go

```diff
@@ -1254,7 +1254,7 @@ func doCompress(cliCtx *cli.Context) error {
 		default:
 		}
 	}
-	if err != nil && !errors.Is(err, io.EOF) {
+	if !errors.Is(err, io.EOF) {
```

Comment:
i did use this command - why did it work?

Reply:
`err != nil` is redundant check. It'll always be the case here...so this doesn't change behavior.